mimic/test/test_core.py

```diff
@@ -63,18 +63,14 @@ def test_from_plugin_includes_all_plugins(self):
             dns_plugin.dns,
             cinder_plugin.cinder
         ))
-        # all plugsin should be on the internal listing
+        # all plugins should be on the internal listing
         self.assertEqual(
             plugin_apis,
             set(core._uuid_to_api_internal.values()))
         # the external listing should still be empty
         self.assertEqual(
             set([]),
             set(core._uuid_to_api_external.values()))
-        self.assertEqual(
-            len(plugin_apis),
-            len(list(core.entries_for_tenant('any_tenant', {},
-                                             'http://mimic'))))
```

Comment:
Q: what was the error?

I'm curious as I put together a lot of the externally hosted API stuff, and am wondering what might be missing for similar support on the internally hosted APIs.

It is well tested, but I'm not familiar enough with the plugin functionality to say that that is equally well tested as the core IAPIMock and IExternalAPIMock functionality.